.github/workflows/google.yml

```diff
@@ -0,0 +1,53 @@
+name: Build and Deploy to Cloud Run
+
+on:
+  release:
+    types: [published]
```

Comment:
**question**: Is this when you push new tag or something? Looks cool

Reply:
yup. there are tons of options for triggering workflows. https://docs.github.com/en/free-pro-team@latest/actions/reference/events-that-trigger-workflows